src/components/modal/report/modal.jsx

```diff
@@ -124,120 +84,93 @@ class ReportModal extends React.Component {
-                                <div className="received">
-                                    <div className="received-header">
-                                        <FormattedMessage id="report.receivedHeader" />
+                            {/* Text input step */}
+                            <FormStep
+                                isWaiting={isWaiting}
+                                nextLabel={{id: 'report.send'}}
+                                onNext={this.handleSubmit}
+                            >
+                                <div className="instructions">
+                                    <div className="instructions-header">
+                                        <FormattedMessage {...category.label} />
                                     </div>
-                                    <FormattedMessage id="report.receivedBody" />
+                                    <FormattedMessage {...category.prompt} />
                                 </div>
-                            ) : (
-                                <div>
-                                    <div className="instructions">
-                                        <FormattedMessage
-                                            id={`report.${type}Instructions`}
-                                            key={`report.${type}Instructions`}
-                                            values={{
-                                                CommunityGuidelinesLink: (
-                                                    <a href="/community_guidelines">
-                                                        <FormattedMessage id="report.CommunityGuidelinesLinkText" />
-                                                    </a>
-                                                )
-                                            }}
-                                        />
+                                <TextArea
+                                    autoFocus
+                                    required
+                                    className="report-text"
+                                    elementWrapperClassName="report-modal-field"
+                                    label={null}
+                                    name="notes"
+                                    validationErrors={{
+                                        isDefaultRequiredValue: intl.formatMessage({id: 'report.textMissing'}),
+                                        maxLength: intl.formatMessage({id: 'report.tooLongError'}),
+                                        minLength: intl.formatMessage({id: 'report.tooShortError'})
+                                    }}
+                                    validations={{
+                                        maxLength: 500,
+                                        minLength: 20
+                                    }}
+                                />
+                            </FormStep>
+
```

Comment:
Tiny quibble -- The other steps don't have an empty line separating them

Reply:
Hmmm they all have an empty line I believe...